lib/package.gi

```diff
@@ -1099,9 +1099,31 @@ InstallGlobalFunction( IsPackageMarkedForLoading, function( name, version )
 ##
 #F  DefaultPackageBannerString( <inforec> )
 ##
-InstallGlobalFunction( DefaultPackageBannerString, function( inforec )
+
+DeclareUserPreference( rec(
+  name:= "ShortBanners",
+  description:= [
+    "If this option is set to <K>true</K>, only the first line of each \
+package banner (including the name, version and description of the package) \
+is printed for packages using the default banner."
```

Comment:
OK, I've thought some more about this, and I wonder if it wouldn't be more useful if this option affected *all* packages. I.e., if it is on, then all packages just get a shortened banner printed when loaded. To implement this, instead of modifying `DefaultPackageBannerString` we could add a separate function, say `ShortPackageBannerString`. And then the code which prints the package banner makes a high-level decision what to do